src/NuGet.Indexing/NuGetSearcherManager.cs

```diff
@@ -183,7 +212,7 @@ protected override NuGetIndexSearcher CreateSearcher(IndexReader reader)
                 // The point of having a specific subclass of the IndexSearcher is that we want to associate a bunch of auxilliary data along
                 // with that specific instance of the reader. The lifetimes are assocaited, hense the inheritance relationship.
 
-                Trace.TraceInformation("about to create a new NuGetIndexSearcher");
+                _logger.LogInformation("about to create a new NuGetIndexSearcher");
```

Comment:
Other logs start with capital. Perhaps for consistency make this one (and a few others) cap as well?